backend/routes/parking.py

```
@@ -0,0 +1,140 @@
+    
+    Returns parking permission status and applicable rules.
+    """
+    log.info(f"Location check request: {request.latitude}, {request.longitude}")
+    
+    try:
+        result = await parking_service.check_location(request)
+        
+        log.info(f"Location check completed: can_park={result.canPark}")
+        return result
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)